src/themes/material/assets/fonts/LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I propose to name it `LICENSE.txt`

Reply:
Done by 5d137009039de5722d0ae6aa11b06297b00f0eac